validator/src/main/java/com/amazon/aoc/helpers/MustacheHelper.java

```diff
@@ -38,14 +39,14 @@
    * @return generated content
    * @throws IOException when the template file is not existed
    */
-  public String render(FileConfig fileConfig, Object dataToInject) throws IOException {
+  public String render(FileConfig fileConfig, Object dataToInject) throws Exception {
     return render(fileConfig.getPath(), dataToInject);
   }
 
-  private String render(String path, Object dataToInject) throws IOException {
+  private String render(URL path, Object dataToInject) throws Exception {
```

Comment:
It's sad to see the exception is less specific. Any thoughts?

Reply:
good point! we should try our best to have specific exceptions defined in most of the cases. will change it back.